webpack/assets/javascripts/react_app/components/AuditsList/ShowOrgsLocs.js

```diff
@@ -4,33 +4,22 @@ import { Col } from 'patternfly-react';
 import ShowTaxonomyInline from './ShowTaxonomyInline';
 import { translate as __ } from '../../common/I18n';
 
-const ShowOrgsLocs = ({ isOrgEnabled, isLocEnabled, orgs, locs }) => (
+const ShowOrgsLocs = ({ orgs, locs }) => (
   <Col sm={10} className="taxonomy-section">
-    {isOrgEnabled && (
-      <ShowTaxonomyInline
-        displayLabel={__('Affected Organizations')}
-        items={orgs}
-      />
-    )}
-    {isLocEnabled && (
-      <ShowTaxonomyInline
-        displayLabel={__('Affected Locations')}
-        items={locs}
-      />
-    )}
+    <ShowTaxonomyInline
+      displayLabel={__('Affected Organizations')}
+      items={orgs}
+    />
+    <ShowTaxonomyInline displayLabel={__('Affected Locations')} items={locs} />
```

Comment:
nitpick: can you indent it as the upper component:
```js
      <ShowTaxonomyInline
        displayLabel={__('Affected Locations')}
        items={locs}
      />
```

Reply:
I wanted to, but eslint yelled at me.